src/persist-client/README.md

```diff
@@ -85,3 +85,49 @@ cargo bench -p mz-persist-client --bench=benches
 Larger writes are expected to take the above latency floor plus however long it
 takes to write the extra data to e.g s3. TODO: Get real numbers for larger
 batches.
+
+## OpenTelemetry Tracing Spans
+
+Persist offers introspection into performance and behavior through integration
+with the [tracing] crate.
+
+[tracing]: https://docs.rs/tracing
+
+Materialize defaults to logs (tracing events) at "info" and above and
+opentelemetry (tracing spans) at "debug" and above. This is because our stderr
+log formatter includes any spans that are active at the time of the log event
+AND that match the log (not opentelemetry) filter. Example of what this looks
+like:
+
+```text
+2022-06-02T21:18:48.220658Z  INFO my_span{my_field=foo}: my_crate::my_module: Hello, world!
```

Comment:
I might steal some of this content for a general-purpose doc in mz someday!

Reply:
feel free!